src/components/SubscriptionBanner.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactElement, useState } from 'react';
+import { SubscriptionsBanner } from './modules/subscriptionsBanner/SubscriptionsBanner';
+import { withKnobs } from '@storybook/addon-knobs';
+import { StorybookWrapper } from '../utils/StorybookWrapper';
+
+export default {
+    component: SubscriptionsBanner,
+    title: 'Components/SubscriptionsBanner',
+    decorators: [withKnobs],
+};
+
+export const defaultStory = (): ReactElement => {
+    const [showBanner, toggleBanner] = useState(true);
```

Comment:
I actually wonder whether this `toggleBanner` state management should take place within the `SubscriptionsBanner` component. EG. If `showBanner` is `true` it returns the component, else it returns `null`.  Moving it up a level into the parent as we're doing at the moment means we'll need to reimplement this across `frontend` and `dotcom-rendering`.

Reply:
Yes, I wasn't sure whether we wanted the banner to own its own dismissal! I'm happy to move it if that will work better for the system though.